pub/js/google-charts.js

```diff
@@ -0,0 +1,151 @@
+(function(){/*
```

Comment:
Updated. However I could find no official number for the version. Typically this would be loaded externally but I decided to pull it in plugin. Their documentation indicates that this is the "new version" 

https://developers.google.com/chart/interactive/docs/release_notes